### ISSUE TITLE
Present the widgets from the readme

### ISSUE THREAD

**Issue by vinyll** [gristlabs/grist-widget#130]
This reposity seems to be a great resource for Grist!

However it's not so obvious to be aware of all the work done here and which component does what.

Some proposals could be either:
- having a list of the widgets with a tiny description
- present a screenshot of the widget within a context
- …

**Comment by paulfitz** [comment]
Yes!! This would be great. I think it would be quite practical in fact to present not just screenshots but live examples, now that [grist-static](https://github.com/gristlabs/grist-static) exists.
